### PR TITLE
refactor!: Remove parent repo libraries

### PR DESCRIPTION
**Pull request by arielvalentin**
Removes any gems and references that belong to the `opentelemetry/opentelemetry-ruby` repository.

Fixes https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues/1

### CODE REVIEW

File: instrumentation/active_job/README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ end
 
 ## Examples
 
-Example usage can be seen in the `./example/active_job.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/instrumentation/active_job/example/active_job.rb)
+Example usage can be seen in the `./example/active_job.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/main/instrumentation/active_job/example/active_job.rb)
```
Comment (robertlaurin) [inline instrumentation/active_job/README.md:35]: Some extra mains in there, was that intentional? Noticing it in a few places.

File: instrumentation/koala/README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ end
 
 ## Examples
 
-Example usage can be seen in the `./example/trace_demonstration.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby/blob/master/instrumentation/koala/example/trace_demonstration.rb)
+Example usage can be seen in the `./example/trace_demonstration.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/master/instrumentation/koala/example/trace_demonstration.rb)
```
Comment (robertlaurin) [inline instrumentation/koala/README.md:36]:
```suggestion
Example usage can be seen in the `./example/trace_demonstration.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/instrumentation/koala/example/trace_demonstration.rb)
```

File: propagator/xray/opentelemetry-propagator-xray.gemspec [hunk 3]
```diff
@@ -37,9 +37,9 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'yard-doctest', '~> 0.1.6'
 
   if spec.respond_to?(:metadata)
-    spec.metadata['changelog_uri'] = "https://open-telemetry.github.io/opentelemetry-ruby/opentelemetry-propagator-xray/v#{OpenTelemetry::Propagator::XRay::VERSION}/file.CHANGELOG.html"
-    spec.metadata['source_code_uri'] = 'https://github.com/open-telemetry/opentelemetry-ruby/tree/master/propagator/xray'
-    spec.metadata['bug_tracker_uri'] = 'https://github.com/open-telemetry/opentelemetry-ruby/issues'
-    spec.metadata['documentation_uri'] = "https://open-telemetry.github.io/opentelemetry-ruby/opentelemetry-propagator-xray/v#{OpenTelemetry::Propagator::XRay::VERSION}"
+    spec.metadata['changelog_uri'] = "https://open-telemetry.github.io/opentelemetry-ruby-contrib/opentelemetry-propagator-xray/v#{OpenTelemetry::Propagator::XRay::VERSION}/file.CHANGELOG.html"
+    spec.metadata['source_code_uri'] = 'https://github.com/open-telemetry/opentelemetry-ruby-contrib/tree/master/propagator/xray'
```
Comment (robertlaurin) [inline propagator/xray/opentelemetry-propagator-xray.gemspec:41]:
```suggestion
    spec.metadata['source_code_uri'] = 'https://github.com/open-telemetry/opentelemetry-ruby-contrib/tree/main/propagator/xray'
```

**Comment by arielvalentin** [comment]
@robertlaurin Pushed up some fixes to deal with CI errors _and_ updated the paths

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,51 +0,0 @@
-# CHANGELOG
```
Comment (plantfansam) [inline CHANGELOG.md:1]: 👋

File: README.md [hunk 5]
```diff
@@ -47,31 +42,46 @@ Maintainers ([@open-telemetry/ruby-maintainers](https://github.com/orgs/open-tel
 This repository contains instrumentation libraries for many popular Ruby
 gems, including Rails, Rack, Sinatra, and others, so you can start
 using OpenTelemetry with minimal changes to your application. See the
-[instrumentation README](instrumentation/) for more.
+[instrumentation README](instrumentation/) for more details.
+
+## Additional Libraries
+
+This repository also contains libraries to aid with interoperablity with vendor specific tracing solutions:
```
Comment (plantfansam) [inline README.md:49]: ✅

File: README.md [hunk 6]
```diff
@@ -47,31 +42,46 @@ Maintainers ([@open-telemetry/ruby-maintainers](https://github.com/orgs/open-tel
...
+
+- [Context Propagation](propagator/): OTTrace and Amazon X-Ray
+- [Resource Detectors](resource_detectors/): Google Cloud Platform
 
 ## Versioning
 
 OpenTelemetry Ruby follows the [versioning and stability document][otel-versioning] in the OpenTelemetry specification. Notably, we adhere to the outlined version numbering exception, which states that experimental signals may have a `0.x` version number.
 
+### Releases
```
Comment (plantfansam) [inline README.md:58]: 👍

File: instrumentation/README.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ Instrumentation libraries provide pre-built OpenTelemetry instrumentation for po
 
 Released instrumentations can be found at the [OpenTelemetry registry](https://opentelemetry.io/registry/?language=ruby&component=instrumentation#).  You can also look in this project's Github repository: Individual instrumentation libraries can be found in subdirectories under `/instrumentation`.
 
-In-progress instrumentations can be found [here](https://github.com/open-telemetry/opentelemetry-ruby/issues?q=is%3Aopen+label%3Ainstrumentation+-label%3A%22help+wanted%22+).
+In-progress instrumentations can be found [here](https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues?q=is%3Aopen+label%3Ainstrumentation+-label%3A%22help+wanted%22+).
```
Comment (plantfansam) [inline instrumentation/README.md:11]: Not a blocker at all, but for dicsussion: we'll need to migrate tickets over from otel-ruby eh? Or do we want to leave them there?

File: instrumentation/pg/test/opentelemetry/instrumentation/pg/instrumentation_test.rb [hunk 8]
```diff
@@ -13,7 +13,7 @@
 
 # This test suite requires a running postgres container and dedicated test container
 # To run tests:
-# 1. Build the opentelemetry/opentelemetry-ruby image
+# 1. Build the opentelemetry/opentelemetry-ruby-contrib image
```
Comment (plantfansam) [inline instrumentation/pg/test/opentelemetry/instrumentation/pg/instrumentation_test.rb:16]: [Checks out](https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/3/files?diff=unified&w=0#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R6) 👍

File: instrumentation/rack/README.md [hunk 9]
```diff
@@ -37,7 +37,7 @@ By default we will set the rack span name to match the format "HTTP #{method}" (
 
 We surface a hook to easily retrieve the rack span within the context of a request so that you can add information to or rename your server span.
 
-This is how the rails controller instrumentation is able to rename the span names to match the controller and action that process the request. See https://github.com/open-telemetry/opentelemetry-ruby/blob/f6fb025bef69f839078748f56516ce38c7d51eb8/instrumentation/action_pack/lib/opentelemetry/instrumentation/action_pack/patches/action_controller/metal.rb#L15-L16 for an example.
+This is how the rails controller instrumentation is able to rename the span names to match the controller and action that process the request. See https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/f6fb025bef69f839078748f56516ce38c7d51eb8/instrumentation/action_pack/lib/opentelemetry/instrumentation/action_pack/patches/action_controller/metal.rb#L15-L16 for an example.
```
Comment (plantfansam) [inline instrumentation/rack/README.md:40]: Not a blocker, but I don't think this commit is avail in this repo: https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/f6fb025bef69f839078748f56516ce38c7d51eb8/instrumentation/action_pack/lib/opentelemetry/instrumentation/action_pack/patches/action_controller/metal.rb#L15-L16. I think this is the right link? https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/instrumentation/action_pack/lib/opentelemetry/instrumentation/action_pack/patches/action_controller/metal.rb#L15-L16

File: instrumentation/bunny/README.md [hunk 10]
```diff
@@ -32,7 +32,7 @@ end
 
 ## Examples
 
-Example usage can be seen in the `./example/bunny.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby/blob/master/instrumentation/bunny/example/bunny.rb)
+Example usage can be seen in the `./example/bunny.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/master/instrumentation/bunny/example/bunny.rb)
```
Comment (robertlaurin) [inline instrumentation/bunny/README.md:35]:
```suggestion
Example usage can be seen in the `./example/bunny.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/instrumentation/bunny/example/bunny.rb)
```

File: instrumentation/bunny/README.md [hunk 11]
```diff
@@ -47,7 +47,7 @@ The `opentelemetry-instrumentation-bunny` gem is distributed under the Apache 2.
 [bunny-home]: https://github.com/ruby-amqp/bunny
 [bundler-home]: https://bundler.io
 [repo-github]: https://github.com/open-telemetry/opentelemetry-ruby
-[license-github]: https://github.com/open-telemetry/opentelemetry-ruby/blob/master/LICENSE
+[license-github]: https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/master/LICENSE
```
Comment (robertlaurin) [inline instrumentation/bunny/README.md:50]:
```suggestion
[license-github]: https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/LICENSE
```

File: instrumentation/http_client/README.md [hunk 12]
```diff
@@ -32,7 +32,7 @@ end
 
 ## Examples
 
-Example usage can be seen in the `./example/trace_demonstration.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby/blob/master/instrumentation/http_client/example/trace_demonstration.rb)
+Example usage can be seen in the `./example/trace_demonstration.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/master/instrumentation/http_client/example/trace_demonstration.rb)
```
Comment (robertlaurin) [inline instrumentation/http_client/README.md:35]:
```suggestion
Example usage can be seen in the `./example/trace_demonstration.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/instrumentation/http_client/example/trace_demonstration.rb)
```

File: instrumentation/http_client/README.md [hunk 13]
```diff
@@ -47,7 +47,7 @@ The `opentelemetry-instrumentation-http_client` gem is distributed under the Apa
 [httpclient-home]: https://github.com/nahi/httpclient
 [bundler-home]: https://bundler.io
 [repo-github]: https://github.com/open-telemetry/opentelemetry-ruby
-[license-github]: https://github.com/open-telemetry/opentelemetry-ruby/blob/master/LICENSE
+[license-github]: https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/master/LICENSE
```
Comment (robertlaurin) [inline instrumentation/http_client/README.md:50]:
```suggestion
[license-github]: https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/LICENSE
```

File: instrumentation/koala/README.md (hunk 2 above)
Comment (robertlaurin) [reply instrumentation/koala/README.md:36]: This got resolved but it's still showing the `/main/master` path.

File: instrumentation/koala/README.md [hunk 14]
```diff
@@ -48,7 +48,7 @@ The `opentelemetry-instrumentation-koala` gem is distributed under the Apache 2.
 [koala-home]: https://github.com/arsduo/koala
 [bundler-home]: https://bundler.io
 [repo-github]: https://github.com/open-telemetry/opentelemetry-ruby
-[license-github]: https://github.com/open-telemetry/opentelemetry-ruby/blob/master/LICENSE
+[license-github]: https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/master/LICENSE
```
Comment (robertlaurin) [inline instrumentation/koala/README.md:51]:
```suggestion
[license-github]: https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/LICENSE
```

File: propagator/xray/opentelemetry-propagator-xray.gemspec (hunk 3 above)
Comment (robertlaurin) [reply propagator/xray/opentelemetry-propagator-xray.gemspec:41]: Also got resolved, but still showing `/master`, I know it was like that but I think nows a good time to correct it. WDYT?

**Comment by robertlaurin** [comment]
Lots of minor comments around paths, do you want to handle that here or as a separate PR @arielvalentin

**Comment by arielvalentin** [comment]
@robertlaurin I would rather fix them here.

File: instrumentation/README.md (hunk 7 above)
Comment (arielvalentin) [reply instrumentation/README.md:11]: We agreed that we would relocate only open issues to `otel-ruby-contrib`

File: instrumentation/rack/README.md (hunk 9 above)
Comment (arielvalentin) [reply instrumentation/rack/README.md:40]:
```suggestion
This is how the rails controller instrumentation is able to rename the span names to match the controller and action that process the request. See https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/instrumentation/action_pack/lib/opentelemetry/instrumentation/action_pack/patches/action_controller/metal.rb#L15-L16 for an example.
```

File: propagator/xray/opentelemetry-propagator-xray.gemspec (hunk 3 above)
Comment (arielvalentin) [reply propagator/xray/opentelemetry-propagator-xray.gemspec:41]: 👍🏼

**Review by robertlaurin — APPROVED** [review]

File: instrumentation/README.md (hunk 7 above)
Comment (plantfansam) [reply instrumentation/README.md:11]: [Notes from SIG](https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit#bookmark=id.50fxhh42vs2w)

**Review by plantfansam — APPROVED** [review]

**Pull request merged**